internal/clients/team3/client.go

```diff
@@ -47,7 +47,7 @@ type client struct {
 	// unused or replaced by getter functions
 	// currentIteration iterationInfo
 	// islandsAlive uint
-	// localPool int
+	localPool float64
```

Comment:
This was removed, as its fetched from the server read handle, pls use the handle instead as that contains the most up to date info

Reply:
It would be great if you could do this for all the uses of shared.Team3

Reply:
Fair enough. I've removed it now.